src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -31,33 +31,79 @@ redirects:
 
 import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening, where it's happening, and what to do about it. New Relic's alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening, where it's happening, and what to do about it. Alerts allow you to define the conditions in your system you want to know about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that proactively notifies your team about potential problems by identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues, and providing a root cause analysis. 
 
-Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that notifies your team about potential problems by identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues, and providing a root cause analysis.
+New Relic's alerts and applied intelligence were designed for busy DevOps teams to end response fatigue: you can detect and diagnose incidents and prioritize them for immediate action. If you haven't already, If you haven't already, [sign up for your free account](https://newrelic.com/signup) to [start receiving alerts](https://one.newrelic.com/alerts-ai) about your data right away!
```

Comment:
I wonder if there's an intermediary step we can add here (or even a 1,2,3 procedure). There's really a step between "signup" and "start alerting."

What about something that instead acknowledges the intervening realities and perhaps builds confidence in the whole process?

e.g.

1. Create an account, if you don't have one already [not sure we need this step]
2. Install one or more agents and integrations [link to product IO]
3. [alerting stuff]

Reply:
Agreed, I've been giving this some thought since our meeting. Going to try to cobble something together in a rework of this and see if I can make a better getting started path.